Tools/Scripts/build-and-collect-pgo-profiles

```diff
@@ -28,18 +48,61 @@ mkdir -p "$BASE/motionmark"
 mkdir -p "$BASE/output"
 mkdir -p "$BASE/Internal/WebKit/WebKitAdditions/Profiling/"
 
-cd ../Internal
+cd Internal
+
+if [[ -z $APP ]] ; then
+    echo "Building WebKit..."
+    rm -rf ../OpenSource/WebKitBuild
```

Comment:
This is going to force a clean build, even for embedded platforms, is that deliberate?

Reply:
That's the existing code. I think that is intentional.

Reply:
Yeah, totally intentional. If your previous build wasn't built with PGO, I wanted to make sure we didn't run into weird bugs.